packages/desktop-client/src/components/budget/MobileTable.tsx

```diff
@@ -1,11 +1,16 @@
-import React from 'react';
+import React, { type CSSProperties, type ReactNode } from 'react';
 
 import { colors } from '../../style';
 import View from '../common/View';
 
 export const ROW_HEIGHT = 50;
 
-export const ListItem = ({ children, style, ...props }) => {
+type ListItemProps = {
+  children?: ReactNode;
+  style: CSSProperties;
+};
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏should also add `ViewProps` for the `props` that got spread and passed to `View` component.
```suggestion
} & ViewProps;
```